.gitattributes

```diff
@@ -1,5 +1,5 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
-* text=auto
+* text eol=lf
```

Comment:
The problem I have with auto is the packing of resources: We want resource files to have \n line endings. If we do a release from a Windows machine, we might get problems.

I don't trust the git auto setting.

If you want to, we can set this explicitly for e.g. java files:
```
.java text
```

I believe that developers use a good text editor that can handle this and not the Windows default editor. So I think that we should stick to \n to prevent any further line ending problems.